Application.py

```diff
@@ -50,49 +52,23 @@
 
     elif(args.role == 'EVSE'):
         with Evse(args.interface_type, args.interface, args.mac) as evse:
-            # Set regulation parameters of the charger
-            evse.getCharger().setEvseDeltaVoltage(0.5)
-            evse.getCharger().setEvseDeltaCurrent(0.05)
-
-            # Set limitations of the charger
-            evse.getCharger().setEvseMaxVoltage(400)
-            evse.getCharger().setEvseMaxCurrent(100)
-            evse.getCharger().setEvseMaxPower(25000)
+            # apply config to evse
+            if config is not None:
+                evse.load(config)
 
             # Start the charger
             evse.getCharger().start()
 
-            # Set the schedule
-            digest_value = range(31)
-            signature_value = range(63)
-            schedule = {
-                "code": 0,
-                "schedule_tuples": [{
-                    'schedule_tuple_id': 1,
-                    'schedules':[
-                        {
-                            "start": 0,
-                            "interval": 1800,
-                            "power": evse.getCharger().getEvseMaxPower()
-                        },
-                        {
-                            "start": 1800,
-                            "interval": 1800,
-                            "power": int(evse.getCharger().getEvseMaxPower() * 0.75)
-                        },
-                        {
-                            "start": 3600,
-                            "interval": 82800,
-                            "power": int(evse.getCharger().getEvseMaxPower() * 0.5)
-                        }
-                    ]
-                }]
-            }
-            evse.setSchedule(schedule)
-
-            # Start the EVSE loop
-            evse.whitebeet.networkConfigSetPortMirrorState(args.portmirror)
-            evse.loop()
-            print("EVSE loop finished")
+            #set the Whitebeet time
+            #evse.setTime()  

```

Comment:
I think this should not be commented out, we need the time for checking the injected certificates.